### PR TITLE
Chore: Adds informative PR body text

### PR DESCRIPTION
**Pull request by hugohaggmark**

### CODE REVIEW

File: update-changelog/index.ts [hunk 1]
```diff
@@ -64,7 +64,7 @@ class UpdateChangelog extends Action {
 
 		await octokit.octokit.pulls.create({
 			base: 'main',
-			body: 'This exciting! So much has changed!',
+			body: 'This exciting! So much has changed!\nDO NOT CHANGE THE TITLES DIRECTLY IN THIS PR, everyting in the PR is auto generated.',
```
Comment (hugohaggmark) [inline update-changelog/index.ts:67]: Not sure if it's `\n` or `<br />`

Comment (ashharrison90) [reply update-changelog/index.ts:67]: think it's `\n`, same as for comments (e.g. https://github.com/grafana/grafana-github-actions/blob/main/backport/backport.ts#L178) 👍

**Review by ashharrison90 — APPROVED** [review]
lgtm 🙌 

curiosity question - what happens if someone does change the PR title? is it just that the next time it's run it can't find the same PR again? 🤔

File: update-changelog/index.ts (hunk 1 above)
Comment (axelavargas) [reply update-changelog/index.ts:67]:
```suggestion
			body: 'This exciting! So much has changed!\nDO NOT CHANGE THE TITLES DIRECTLY IN THIS PR, everything in the PR is auto-generated.',
```

**Review by axelavargas — APPROVED** [review]
👏🏾 I left only a minor suggestion :)

**Comment by hugohaggmark** [comment]
> lgtm 🙌
> 
> curiosity question - what happens if someone does change the PR title? is it just that the next time it's run it can't find the same PR again? 🤔

I guees the next time the update change log action is run the PR title is changed, or maybe I don't understand your question.

**Pull request merged**